test/test_sorted_containers.jl

```diff
@@ -1765,3 +2093,4 @@ end
         end
     end
 end
+
```

Comment:
can we label the new tests with a comment including the URL of the github issues that they prove is solved?